src/impl_x86__base_implementation.inl

```diff
@@ -406,8 +406,11 @@ X86Info GetX86Info(void) {
       IsVendor(leaves.leaf_0, CPU_FEATURES_VENDOR_AUTHENTIC_AMD);
   const bool is_hygon =
       IsVendor(leaves.leaf_0, CPU_FEATURES_VENDOR_HYGON_GENUINE);
+  const bool is_zx = 
```

Comment:
`is_zhaoxin` please (`is_zx` is too short.)

Reply:
Yes. I have changed "is_zx" to "is_zhaoxin".